### PR TITLE
Robot_Framework_Goibibo.com

### PR DESCRIPTION
**Pull request by Subhajit908**
Hi All,

Please review my Robot Framework for Goibibo.com.

### CODE REVIEW

File: Subhajit/Robot_Framework/Result/log.html
Comment (prateek1604) [inline Subhajit/Robot_Framework/Result/log.html:1]: Please remove all log, report, result files from PR

File: Subhajit/Robot_Framework/Tests/Goibibo.robot [hunk 1]
```diff
@@ -0,0 +1,10 @@
+*** Settings ***
+Documentation       This Test Suite file contains all TC;s related to Goibibo
+Resource            ../Resources/Goibibo_Keywords.robot
+
+*** Test Cases ***
+Validate Flight Search
+    Given User nevigate to Goibibo website
+    Then User click on Flight Tab
```
Comment (prateek1604) [inline Subhajit/Robot_Framework/Tests/Goibibo.robot:8]: Can we have a when statement in between....

File: Subhajit/Robot_Framework/Tests/Goibibo.robot [hunk 2]
```diff
@@ -0,0 +1,10 @@
+*** Settings ***
+Documentation       This Test Suite file contains all TC;s related to Goibibo
+Resource            ../Resources/Goibibo_Keywords.robot
+
+*** Test Cases ***
+Validate Flight Search
+    Given User nevigate to Goibibo website
```
Comment (prateek1604) [inline Subhajit/Robot_Framework/Tests/Goibibo.robot:7]: can you please correct the spelling of navigate

File: Subhajit/Robot_Framework/Resources/Goibibo_Keywords.robot [hunk 3]
```diff
@@ -0,0 +1,25 @@
+*** Settings ***
+Library     SeleniumLibrary
+
+*** Keywords ***
+User nevigate to Goibibo website
+    open browser    https://www.goibibo.com/        chrome
```
Comment (prateek1604) [inline Subhajit/Robot_Framework/Resources/Goibibo_Keywords.robot:6]: This URL `https://www.goibibo.com/` can be different for different env's, So we should parametrize and parse it from configuration Env related files

**Review by prateek1604 — CHANGES_REQUESTED** [review]
@Subhajit908 Can you please check the comments and let me know if any help is required

**Review by komalkapde — APPROVED** [review]

**Review by komalkapde — APPROVED** [review]

**Review by prateek1604 — APPROVED** [review]

**Review by pcube0608 — APPROVED** [review]

**Pull request merged**